git-tar/function/handler.go

```diff
@@ -26,6 +27,9 @@ func Handle(req []byte) []byte {
 		os.Exit(-1)
 	}
 
+	// force github username to lowercase to prevent issues downstream
+	pushEvent.Repository.Owner.Login = strings.ToLower(pushEvent.Repository.Owner.Login)
```

Comment:
Not sure we should mutate the value from GitHub, this also invalidates the SHA signature.